packages/@okta/vuepress-site/docs/release-notes/2020-09-4/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2020.09.4
+
+| Change                                                    | Expected in Preview Orgs |
+| --------------------------------------------------------- | ------------------------ |
+| [Bugs fixed in 2020.09.4](#bug-fixed-in-2020-09-4)       | September 30, 2020           |
```

Comment:
Was the release date pushed to today?

Reply:
No it rolled out last night, but at 11 pm so I didn't get to it until now. Technically yesterday's date is correct.